Lib/test/_test_multiprocessing.py

```diff
@@ -6045,6 +6045,21 @@ def test_resource_tracker_exit_code(self):
                     cleanup=cleanup,
                 )
 
```

Comment:
I suspect we need something like this for some of the buildbots:
```suggestion
    @unittest.skipUnless(hasattr(signal, pthread_sigmask), "pthread_sigmask is not available")
```